pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>no.idporten.test.generate</groupId>
+    <artifactId>idporten-fnr-generate</artifactId>
+    <version>DEV-SNAPSHOT</version>
+    <name>idporten-fnr-generate</name>
+    <description>Library for ID-porten generate test fodselsnummer</description>
+
+    <properties>
+        <idporten-validators-version>0.0.4</idporten-validators-version>
+        <junit-jupiter-version>5.8.2</junit-jupiter-version>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
+    </properties>
```

Comment:
Jeg tror du vil få større gjenbruk i applikasjoners tester dersom Java 8 brukes?

Ser for meg at dette biblioteket kan bli fint å bruke i enhetstester.  Sikre at vi ikke bruker så statiske data (gi litt variasjon) samt å sikre at vi ikke bruker relle fnr/dnr.

Reply:
Eg endrar til java11. ikkje så lyst å gå ned heilt på 8... :)